packages/devextreme/js/__internal/ui/m_file_uploader.ts

```diff
@@ -60,6 +60,16 @@ const FILEUPLOADER_AFTER_LOAD_DELAY = 400;
 const FILEUPLOADER_CHUNK_META_DATA_NAME = 'chunkMetadata';
 const DRAG_EVENT_DELTA = 1;
 
+const DIALOG_TRIGGER_EVENT_NAMESPACE = 'dxFileUploaderDialogTrigger';
+
+const keyUpEventName = 'keyup';
+
+const dialogTriggerClickEventName = addNamespace(clickEventName, DIALOG_TRIGGER_EVENT_NAMESPACE);
+const dialogTriggerKeyUpEventName = addNamespace(keyUpEventName, DIALOG_TRIGGER_EVENT_NAMESPACE);
```

Comment:
Why do you create it in a global scope if it's used in the single place? Please move it to the appropriate method